Source/WebCore/PAL/pal/spi/mac/NSSpellCheckerSPI.h

```diff
@@ -43,6 +43,12 @@ extern NSString *NSTextCheckingSuppressInitialCapitalizationKey;
 
 @end
 
+#endif // USE(APPLE_INTERNAL_SDK)
+
+#if HAVE(NSSPELLCHECKER_CORRECTION_INDICATOR_UNDERLINE_COLOR)
+@interface NSSpellChecker (Staging_105286196)
```

Comment:
If you haven't, can you file a bug for removing this staging code?

Reply:
Filed a bug and added a comment.